runtime/compiler/control/J9Recompilation.cpp

```diff
@@ -561,10 +561,9 @@ J9::Recompilation::getCounterSymRef()
    }
 
 
-/////////////////////////
+
```

Comment:
It looks like this line of just slashes was accidentally deleted.  May I ask you to add it back?

```
/////////////////////////
```